matcher/solvers/perturbed_maximization_solver.py

```diff
@@ -0,0 +1,550 @@
+"""
+A paper-reviewer assignment solver that aims to trade off between the total
+affinity score and the randomness of the assignment. The solver is based on
+the algorithm described in Xu et al 2023.
+
+The solver relies on the Gurobi optimizer to solve convex quadratic programs
+that arise in the assignment problem, and the CFFI library to interface with
+a sampling program written in C.
+"""
+
+import logging
+import numpy as np
+import gurobipy as gp
+from cffi import FFI
+from .core import SolverException
+from .bvn_extension import run_bvn
+from .minmax_solver import MinMaxSolver
+
+class PerturbedMaximizationSolver:
```

Comment:
can you rename the class as "PerturbedMaximization" with no solver class? this way matches with the format of all the other solvers `['MinMax', 'FairFlow', 'Randomized', 'FairSequence']`

Reply:
ohhh you are right, ignore my comment then.